src/Shouldly.Shared/ShouldlyExtensionMethods/ShouldBeEnumExtensions.cs

```diff
@@ -0,0 +1,85 @@
+﻿#define PORTABLE
```

Comment:
I don't know how to switch profiles, that's why I did the define.

I'm sure there's an easy way, can you enlighten me? :)